apps/ftu/test/unit/app_test.js

```diff
@@ -1,78 +0,0 @@
-/* global MocksHelper, MockNavigatorMozMobileConnections, MockL10n,
```

Comment:
Yeah - it feels like it may be more useful to actually keep this around, so I've refactored the test to just perform a very basic sanity check of AppManager.init - just checking that we trigger an event. Feel free to remove this file if you want in a follow-up though.